src/client/actions/movies/index.js

```diff
@@ -46,14 +39,23 @@ export function updateSortBy(sortBy) {
         type: UPDATE_SORT_BY,
         payload: sortBy
     };
-};
+}
 
 export function updateSortByAsync(sortBy) {
     return function (dispatch) {
         dispatch(updateSortBy(sortBy));
         return Promise.resolve();
     }
-};
+}
+
+export function resetMoviesSearch() {
+    return function (dispatch) {
+        dispatch(updateSearchTerm(''));
+        dispatch(updateSortBy(SORT_BY_DEFAULT_VALUE));
+        dispatch(updateSearchBy(SEARCH_BY_DEFAULT_VALUE));
+        dispatch(showMovies({data: {data: []}}));
```

Comment:
Just advice, all this actions can be applied in scope of one action in the reducer.